community/codestar/custom-ci-cd-pipeline/README.md

```diff
@@ -0,0 +1,45 @@
+# custom-ci-cd-pipeline
+
+This template creates a custom CodePipeline pipeline for continuous integration and continuous delivery, for *two environments* named `staging` and `production`.
```

Comment:
Ok, don't laugh -- but can you update the README.md to specify that this requires an **existing** CodeCommit repo as the source via Parameter? I thought it would generate the repo as part of the pipeline.

Reply:
Hi @cmmeyer 👋

Sure, no problem. I've updated the `README.md` file by adding a `CodeCommit repository` section.